packager/hls/base/media_playlist.h

```diff
@@ -268,6 +273,8 @@ class MediaPlaylist {
   bool target_duration_set_ = false;
   uint32_t target_duration_ = 0;
 
+  double start_timestamp = 0;
```

Comment:
Ends with "_" to be consistent. Can you also add a comment explaining what it is?

Reply:
Ping.

Reply:
Done.